fixtures/datagen/templates/no_cert.json

```diff
@@ -47,42 +47,9 @@
         "country_of_registration": ""
     },
     "contacts": {
-        "technical": {
-            "name": "",
-            "email": "",
-            "phone": "",
-            "person": null,
-            "extra": {
-                "@type": "type.googleapis.com/gds.models.v1.GDSContactExtraData",
-                "verified": false,
-                "token": "",
-                "email_log": []
-            }
-        },
-        "administrative": {
-            "name": "",
-            "email": "",
-            "phone": "",
-            "person": null,
-            "extra": {
-                "@type": "type.googleapis.com/gds.models.v1.GDSContactExtraData",
-                "verified": false,
-                "token": "administrative_token",
-                "email_log": []
-            }
-        },
-        "legal": {
-            "name": "",
-            "email": "",
-            "phone": "",
-            "person": null,
-            "extra": {
-                "@type": "type.googleapis.com/gds.models.v1.GDSContactExtraData",
-                "verified": false,
-                "token": "legal_token",
-                "email_log": []
-            }
-        },
+        "technical": null,
+        "administrative": null,
+        "legal": null,
```

Comment:
Changed so that the fixtures work with TRISA validation (it expects contacts which are not nil to have an email address).